client/task-list/tasks/payments/index.js

```diff
@@ -383,6 +388,7 @@ export default compose(
 			'woocommerce_eway_settings',
 			'woocommerce_razorpay_settings',
 			'woocommerce_mollie_payments_settings',
+			'woocommerce_payubiz_settings',
```

Comment:
Not your original code, so no worries if you keep it this way (and not sure why I didn't notice till now).
We should have a added config with dependent options (`options_needed: ['woocommerce_payubiz_settings']` some have more then one), so that here we could just loop through all the `methods` and not have to worry about adding one for each new payment method.

Reply:
Ah yep, I'm gonna leave this one for this PR, mainly because I think a better treatment would be a proper refactor of this code.